code/datums/action.dm

```diff
@@ -19,6 +19,7 @@
 	var/mob/owner
 
 /datum/action/New(Target)
+	START_PROCESSING(SSobj, src)
```

Comment:
then it needs to only be on umbrage abilities and ONLY updated when they gain/lose psi.